testing_and_setup/compass/utility_scripts/LIGHTparticles/make_particle_file.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
+"""
+File writes a particle input dataset for use in MPAS-O / E3SM.
+
+Phillip J. Wolfram
+Last Modified: 07/12/2018
+"""
+
+import netCDF4
+import numpy as np
+
+verticaltreatments = {'indexLevel':1, 'fixedZLevel': 2, 'passiveFloat': 3, 'buoyancySurface': 4, 'argoFloat': 5}
+defaults = {'dt': 300, 'resettime': 1.0*24.0*60.0*60.0}
+
+def use_defaults(name, val):
+    if (val is not None) or (val is not np.nan):
+        return val
+    else:
+        return defaults[name]
+
+def ensure_shape(start, new):
+    if isinstance(new, (int, float)):
+        new = new*np.ones_like(start)
+    return new
+
+def southern_ocean_only_xyz(x, y, z, maxNorth=-45.0):
```

Comment:
@bradyrx, I've updated this script to do the filtering via something like `./make_particle_file.py -i /home/ccsm-data/inputdata/ocn/mpas-o/oQU240/ocean.QU.240km.151209.nc -g /home/ccsm-data/inputdata/ocn/mpas-o/oQU240/mpas-o.graph.info.151209.part. -p 72 -o /lcrc/group/acme/pwolfram/acme_scratch/anvil/20180626.GMPAS-IAF.T62_oQU240.anvil.particles/run/particles.nc --spatialfilter SouthernOceanXYZ -t ['passive','surface']` (on blues for me).

Note, I've "hard-coded" the option `maxNorth=-45.0` for now.  I tested by completing a quick G-case 240km run over 30 days.